content/blog/2021-11-30-international-rse-survey.md

```diff
@@ -0,0 +1,38 @@
++++
+title = "Take the 2021 international RSE survey: help us improve the RSE community"
+
+[extra]
+
+author = "Patric Holmvall, Radovan Bast, Richard Darst" 
++++
+
+
+## Take the survey
+
+Do you develop research codes, or generally work with research software or data analysis? Perhaps you even feel more connected to the technology side: programming, software, data, etc., rather than the writing of papers? Although this work is crucial to the research, the modern research environment seldom fully recognizes it, in contrast to other methodology development. Take part in the [international Research Software Engineer (RSE) survey](https://softwaresaved.limequery.com/386272?lang=en) to help us understand how we can improve the situation. Your current position or job title does not matter, only that you feel connected to software, data, computation, or similar technology.
+
+Link to the survey: <https://softwaresaved.limequery.com/386272?lang=en>
+
+
+## You can also help us by sharing the survey with others
+
+Perhaps you are not writing research software as part of your work, but maybe your colleagues or collaborators are? Or perhaps a research group or research institute that you are in contact with? You can help us and the reach of the survey immensely, by sharing and making the survey more visible in mailing lists, group chats, blog posts, or newsletters. There are many RSEs out there who may not know about the RSE community, and that there are many more RSEs working across the Nordics/Baltics. Many are not even aware about the term ["Research Software Engineer"](https://www.software.ac.uk/blog/2016-08-17-not-so-brief-history-research-software-engineers-0), and that they are one. This survey is one way to reach them, and to raise awareness.
+
+
+## What and who is an "RSE"?
+
+A [Research Software Engineer](https://www.software.ac.uk/blog/2016-08-17-not-so-brief-history-research-software-engineers-0) is a person who combines the mindset of a researcher with the skills of a software developer. We interpret this inclusively, for people who might consider themselves data engineers, system specialists, and so on. For decades, research has been dependent on such roles, but without providing them proper career support or recognition. The Research Software Engineer movement aims to improve this situation by representation and advocating for these people and their work.
+
+
+## RSEs in the Nordics and Baltics
+
+[Nordic-RSE](https://nordic-rse.org/) is the Nordic+Baltics organization advocating for RSEs. Founded in 2017 and registered in Finland in 2021, it provides a [community](https://coderefinery.zulipchat.com), [periodic seminars](https://nordic-rse.org/events/seminar-series/), and in the future [other events](https://nordic-rse.org/events/) which can connect RSEs to like-minded individuals to promote career development. It will also advocate for the value of RSE services within the research process.
```

Comment:
```suggestion
[Nordic-RSE](https://nordic-rse.org/) is the Nordic+Baltics association advocating for RSEs. Founded in 2017 and registered in Finland in 2021, it provides a [community](https://coderefinery.zulipchat.com), [periodic seminars](https://nordic-rse.org/events/seminar-series/), and in the future [other events](https://nordic-rse.org/events/) which can connect RSEs to like-minded individuals to promote career development. It will also advocate for the value of RSE services within the research process.
```

Reply:
?

Reply:
Sounds good. Cannot commit suggestion for some reason. Will include it manually in an upcoming revision. Thanks for your feedback!